jmx_prometheus_javaagent/src/main/java/io/prometheus/jmx/Arguments.java

```
@@ -0,0 +1,150 @@
+
+        Mode mode;
+        String host = null;
+        Integer port = null;
+        String filename;
+
+        if (matcher.matches()) {
+            switch (matcher.groupCount()) {
+                case 2:
+                    {
+                        mode = Mode.HTTP;
+                        host = DEFAULT_HOST;
+
+                        try {
+                            port = Integer.parseInt(matcher.group(1));
+                        } catch (NumberFormatException e) {
+                            throw new ConfigurationException(
+                                    format("Malformed arguments [%s]", agentArgument));
+                        }
+                        filename = matcher.group(2);
+                        break;
+                    }
+                case 3:
+                    {
+                        mode = Mode.HTTP;
+                        host = matcher.group(1) != null ? matcher.group(1) : DEFAULT_HOST;
+
+                        if (host.startsWith("[") && host.endsWith("]") && host.length() > 3) {
+                            host = host.substring(1, host.length() - 1);
+                        }
+
+                        port = Integer.parseInt(matcher.group(2));
+                        filename = matcher.group(3);
+                        break;
+                    }
+                default:
+                    {
+                        throw new ConfigurationException(
+                                format(
+                                        "Malformed arguments for Standalone HTTP mode [%s]",
+                                        agentArgument));
+                    }
+            }
+
+            if (host.trim().isEmpty()) {
+                throw new ConfigurationException(
+                        format("Malformed arguments for Standalone HTTP mode [%s]", agentArgument));
+            }
+        } else {
+            mode = Mode.OPEN_TELEMETRY;
```

If I understand correctly, if neither a port for the Prometheus HTTP endpoint nor `opentelemetry:` in the config is specified, `jmx_exporter` will silently do nothing.

Should we print an error and exit in that case?